### DISCUSSION TITLE
Set globalTypes default value on story level?

### DISCUSSION THREAD

**Discussion by magnattic** [storybookjs/storybook#19538] [Help]
I have the following `globalTypes` to enable a toolbar in storybook that lets me select the theme:

```
export const globalTypes = {
  theme: {
    name: 'Theme',
    description: 'Global UrbyoTheme',
    defaultValue: UrbyoTheme.Light,
    toolbar: {
      icon: 'mirror',
      items: [UrbyoTheme.Light, UrbyoTheme.Dark],
      showName: true,
      dynamicTitle: true,
    },
  },
};
```

This works fine and I can switch the theme through the toolbar: 
![image](https://user-images.githubusercontent.com/7560166/196660997-d10ee1fc-657d-441c-b353-afc845a43e8a.png)

**What I would like to do now is to switch the default theme selected for a certain story. How can I do this?**

What I tried (which did not work):

```
export default {
  title: 'Components/Calculator',
  component: CalculatorUI,
  parameters: {
    theme: {
      default: UrbyoTheme.Dark,
    },
  },
};
```

The docs don't mention this use case, is it possible?

**Comment by fredericrous** [comment]
for anyone coming here from google, it's not possible at this time (Jun 28, 2024). It should become possible to set globalTypes default value on story level tho: https://github.com/storybookjs/storybook/discussions/23347
